.github/CODEOWNERS

```diff
@@ -1,8 +1,19 @@
 # Last match in file takes precedence.
 
-/Content.*/SimpleStation14/ @DEATHB4DEFEAT
+# C# code
+/Content.*/ @DeltaV-Station/maintainers
 
-/Resources/*.yml @Colin-Tel
-/Resources/*/SimpleStation14/ @DEATHB4DEFEAT
-/Resources/Maps/ @IamVelcroboy
-/Resources/Prototypes/Maps/ @IamVelcroboy
+# YML files
+/Resources/**.yml @DeltaV-Station/yaml-maintainers
+
+# Sprites
+/Resources/Textures/ @IamVelcroboy
+
+# Lobby art and music - automatically direction issues since its immediately visible to players
+/Resources/Audio/Lobby/ @DeltaV-Station/game-directors
+/Resources/Textures/LobbyScreens/ @DeltaV-Station/game-directors
```

Comment:
According to Github, `@DeltaV-Station/game-directors` does not exist.

Reply:
![04:07:38](https://github.com/user-attachments/assets/d3f8f6b2-8ac9-4f5f-80ec-f5ba22acb8ed)

Reply:
Not sure why the code is flagged as erroneous during review, then. Ah well.